### ISSUE TITLE
Cannot upload photos with certain account

### ISSUE THREAD

**Issue by Victor-Morel** [pixelfed/support#2]
Hello,
I set up an instance of pixelfed, which worked smoothly the first months.
But I'm now having issue uploading any (I tried with different images) photo: clicking the button ``Create a post'' does not result in anything.
I still can with other accounts, but the most used one cannot anymore.
I can't seem to find any help in my apache2 logs, and the /storage/logs/laravel.log are of no use.

I'm not sure whether it's a bug or a misconfiguration, any help would be welcomed!

Thanks for developing the software in any case

**Comment by Victor-Morel** [comment]
I solved the pdo issue, but it didn't solve my upload problem.
php -m returns a list with the extensions required in [here](https://docs.pixelfed.org/installing-pixelfed/prerequisites.html#php).

I'm still able to upload with certain accounts, but not all.
My installation of Pixelfed was and still is able to communicate with the database (hence my lack of energy put in the pdo-mysl warning)

**Comment by Victor-Morel** [comment]
I continued my investigation.
The warning related to PDO was most likely due to an issue in the declaration of the extension (cf this [stackoverflow post](https://stackoverflow.com/questions/55027457/laravel-php-7-3-pdo-mysql-reports-missing-driver-possible-due-to-undefined-symb)).

The upload issue started again with a newly created account, seemingly because of specific metadata: the pictures causing the problem originate from the same batch as the one creating the problem in the first place. 
It may be linked to wrong interpretation of the orientation of pictures, between portrait and landscape, crashing the upload for the account.
I'll conduct further tests as soon as I can.

Do you know where could I find logs pertaining to Pixelfed errors?

**Comment by trwnh** [comment]
Can you provide the original images for testing uploads to other instances? If it is a bug in Pixelfed it should fail there too.

Logs are kept in `storage/logs/laravel.log`, and if you are running a systemd service, you may be able to see some relevant logged info by running `systemctl status pixelfed-service-name` and `journalctl -u pixelfed-service-name`

**Comment by Victor-Morel** [comment]
After further tests, I determined that the issue was caused by a faulty php configuration: I had only  `post_max_size` set at the desired value, while `upload_max_filesize` had been left at its original value. 
It resulted in upload failures for pictures larger than specified by `upload_max_filesize`.

The problem is therefore mostly solved.
I say mostly because the original account is definitely broken: I cannot upload any picture (even pics smaller than the previous `upload_max_filesize`).
While errors on the `laravel.log` put me on the track, the broken account doesn't issue anything.

Is it possible to debug this account?
